client/client-multi/client-c/test/test_kaa_channel_manager.c

```diff
@@ -458,7 +464,24 @@ void test_get_bootstrap_client_sync_serialize(void **state)
     kaa_channel_manager_destroy(channel_manager);
 }
 
+void test_process_auth_failure(void **state)
+{
+    (void)state;
+
+    kaa_error_t error_code;
+    kaa_channel_manager_t *channel_manager = NULL;
+    kaa_auth_failure_reason reason;
+
+    error_code = kaa_channel_manager_create(&channel_manager, &kaa_context);
+    assert_return_code(error_code, KAA_ERR_NONE);
```

Comment:
You misunderstood `assert_return_code()' purpose. Please, check [documentation](https://api.cmocka.org/group__cmocka__asserts.html#ga9370e705f22fa98b18664bacb9c525e8). Generally, `assert_int_equal` should be used.